### ISSUE TITLE
Missing support for negative zero

### ISSUE THREAD

**Issue by Janpot** [blitz-js/superjson#83]
Came across this while evaluating this library. It looks like it supports `Infinity` and `NaN` (not sure if it's an oversight, but these are not included in the table in the readme), but it doesn't seem to support serializing negative 0.

```js
const input = 1 / -Infinity; // -0
console.log(1 / input);
// -Infinity
console.log(1 / superjson.parse(superjson.stringify(input)));
// Infinity
```

**Comment by KATT** [comment]
Just of curiosity, when would you ever use a negative zero?

Personally I've never intentionally had use of negative zero, infinity, & NaN and prefer if a library throws whenever I encounter those as it's always just represented a hidden bug.

**Comment by Skn0tt** [comment]
That‘s interesting! We should support this, I guess. Would you be open to provide a PR? I think it should be solvable by adding a new transformer in `transformers.ts`.

**Comment by Janpot** [comment]
>Would you be open to provide a PR?

I'm sorry, I'm not planning to use this library in the near future. just thought I'd report an effort case iInoticed.

**Comment by Skn0tt** [comment]
No worries! Then this issue is up for grabs. Maybe someone else wants to take a shot at it :)

**Comment by Skn0tt** [comment]
Closed by #84
